src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IncentiveSummaryService.kt

```diff
@@ -41,21 +41,22 @@ class IncentiveSummaryService(
 
     val iepLevelsDeferred = async { iepLevelService.getIepLevelsForPrison(prisonId) }
 
-    val offenderNos = prisoners.map(OffenderSearchPrisoner::prisonerNumber)
-    val positiveCaseNotes = async { getCaseNoteUsage("POS", "IEP_ENC", offenderNos) }
-    val negativeCaseNotes = async { getCaseNoteUsage("NEG", "IEP_WARN", offenderNos) }
-
     val bookingIds = prisoners.map(OffenderSearchPrisoner::bookingId)
     val provenAdjudications = async { getProvenAdjudications(bookingIds) }
-    val iepDetails = getIEPDetails(bookingIds)
 
-    val positiveCount = positiveCaseNotes.await()
-    val negativeCount = negativeCaseNotes.await()
+    val reviews = prisonerIepLevelRepository.findAllByBookingIdInOrderByReviewTimeDesc(bookingIds = bookingIds).toList()
+    val iepDetails = getCurrentAndHistoricalReviews(reviews).associateBy(IepResult::bookingId)
+
+    val deferredBehaviourCaseNotesSinceLastReview = async {
+      behaviourService.getBehaviours(prisonerIepLevelRepository.findAllByBookingIdInOrderByReviewTimeDesc(bookingIds = bookingIds).toList())
```

Comment:
I may miss something but you could reuse the `reviews` value here AFAICT should contain the same list so we can save the trip to the DB.

Reply:
Duh, I specifically split this out so I could do that, must have been a cut and paste job when refactoring!  Will change